src/targets/gpu/include/migraphx/gpu/context.hpp

```diff
@@ -126,6 +127,41 @@ struct hip_device
         }
 #endif
 
+#if MIGRAPHX_USE_HIPBLASLT
+        auto get_hipblaslt()
+        {
+            setup();
+            if(hblthandle == nullptr)
+            {
+                hblthandle = create_hipblaslt_handle_ptr(get());
+            }
+            assert(hblthandle.get() != nullptr);
+            return hblthandle.get();
+        }
+
+        auto get_hipblaslt_preference()
+        {
+            setup();
+            if(hbltpreference == nullptr)
+            {
+                hbltpreference = create_hipblaslt_preference_ptr();
+            }
+            assert(hbltpreference.get() != nullptr);
+            return hbltpreference.get();
```

Comment:
The preference pointer should be stored in the operator and created either during compile_blas or finalize.

Reply:
Actually it should just be created in the solution class. anyways.

Reply:
Done.